public/components/common/toast/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useOpenSearchDashboards } from '../../../../../../src/plugins/opensearch_dashboards_react/public';
+import { ToastInputFields } from '../../../../../../src/core/public';
+import { ObservabilityAppServices } from '../../../../common/types/shared';
+
+type Color = 'success' | 'primary' | 'warning' | 'danger' | undefined;
+
+export const useToast = () => {
+  const {
+    services: { toasts },
+  } = useOpenSearchDashboards<ObservabilityAppServices>();
+
+  const setToast = (title: string, color: Color = 'success', text?, side?: string) => {
```

Comment:
We can remove the side parameter, if not being used.

Reply:
Side is used when toast during flyout.   So unless toast on right-side is OK while visualization flyout and others are active, only then could we remove it.

Side is being passed-through on OSD API

Reply:
I may be missing something here. Can you please explain how are you using/passing down the side parameter in this function?